ideascube/mediacenter/models.py

```diff
@@ -60,7 +60,7 @@ class Document(SearchMixin, TimeStampedModel):
                                 upload_to='mediacenter/preview',
                                 max_length=10240,
                                 blank=True)
-    credits = models.CharField(verbose_name=_('credit'), max_length=300)
+    credits = models.CharField(verbose_name=_('autorship'), max_length=300)
```

Comment:
It's spelled "authorship".

This is a user-visible string, and it's also visible by translators, so it's important to get it right.